src/dvclive/dvc.py

```diff
@@ -166,3 +166,25 @@ def ensure_dir_is_tracked(directory: str, dvc_repo: "Repo") -> None:
     ]
     if paths_to_track:
         dvc_repo.scm.add(paths_to_track)
+
+
+def get_dvc_stage_template(live):
+    stage = {
+        "cmd": "<python my_code_file.py my_args>",
+        "deps": ["<my_code_file.py>"],
+    }
+    outs = []
+    rel_path = Path(os.path.relpath(os.getcwd(), live._dvc_repo.root_dir))
```

Comment:
I haven't thought it through but the `getcwd()` feels potentially problematic. Maybe we can do it relative to the parent dir of `Live.dir`?